_data/people/ArturU043.yml

```
@@ -0,0 +1,11 @@
+active: true
+focus-area:
+  - as
+  - doma
+institution: University of Washington, Seatle
```

```suggestion
institution: University of Washington
```

To standardize institution names on @gordonwatts (also removes typo of city name)

https://github.com/iris-hep/iris-hep.github.io/blob/2bd6b08b8a0fe935b7008822d2121eafe12bf665/_data/people/gordonwatts.yml#L6